package.json

```diff
@@ -22,6 +28,8 @@
   "workspaces": [
     "packages/css",
     "packages/react",
+    "packages/look-and-feel/css",
+    "packages/look-and-feel/react",
```

Comment:
pourquoi créer un sous-sous package ?

N'aurait-il pas été plus simple de renommer les package css et react en css-agent et react-agent et de créer deux nouveaux package css-client et react-client.

De plus avec ce nommage il nous serait possible de créer des packages css-base et/ou react-base pour des asset commun au 2 design-system

Reply:
Hello,

Merci pour ton review ! Peut-être que j'ai mal compris mais si la structure que tu proposes est la suivante :
```
/
|-- packages/
    |-- css-agent/
    |-- css-client/
    |-- react-agent/
    |-- react-client/
```
alors oui cela aurait été plus simple de renommer les packages mais on ne matérialiserait pas la séparation entre Slash et Look and Feel avec cette structuration applanie.

De plus `-agent` et `-client` ne reflètent pas explicitement `slash` et `look-and-feel` et cela pourrait induire en erreur ceux qui arrivent sur le repo et qui ne connaissent pas nos design systems, qui pourraient penser que nous n'en avons qu'un, décliné en populations cibles, ce qui n'est pas le cas : nous en avons bien 2 (au moins). Pour moi cette notion de "population cible" (agent/client) pourrait être traduite dans le readme de chaque design system si nécessaire.

Je suis donc d'avis à garder 2 répertoires distincts et clairement nommés du nom de chaque DS, conformément à ce qui a été décidé.

Par contre @PeioDeLaVega il n'est pas nécessaire ici de nommer chaque sous répertoire, tu peux faire
```
+    "packages/look-and-feel/*"
```

Reply:
Pour la structure oui c'est bien ça dont je parlait
```
/
|-- packages/
    |-- css-agent/
    |-- css-client/
    |-- react-agent/
    |-- react-client/
```

Personnellement je trouve cette structure plus simples et la séparation est bien visible mais ce n'est que mon avis 😉

Après pour le nommage depuis le début on est parti sur `agent` pour désigner `slash` et `client` pour `look-and-feel`

Cela risque de faire beaucoup de changement pour les contributeurs.

à la limite ce que l'on peux faire c'est
```
/
|-- packages/
    |-- look-and-feel-css/
    |-- look-and-feel-react/
    |-- slash-css/
    |-- slash-react/
```
on reste sur une structure à plat des dossiers mais comme le nom du design-system et en préfix cela permet une meilleur visibilité.

Reply:
> Après pour le nommage depuis le début on est parti sur agent pour désigner slash et client pour look-and-feel
> 
> Cela risque de faire beaucoup de changement pour les contributeurs.

Yes ça fait beaucoup de changements j'en suis consient, en même temps je trouve normal de faire des ajustements en début de projet. Le risque serait de lier l'avenir du projet à des positions initiales qu'on se refuse de revoir. 

En l'occurrence "agent" ou "client" n'ont pas plus de sens que "B2B" ou "B2C" par exemple, le fait est qu'ils s'appellent "Slash" et "Look and Feel" : les obfusquer par "agent", "client" (ou même "B2B" et "B2C" pour reprendre mon exemple) ne servirait qu'à induire la personne qui n'a pas le contexte à penser que nous n'avons qu'un design system !

> à la limite ce que l'on peux faire c'est
> 
> /
> |-- packages/
>     |-- look-and-feel-css/
>     |-- look-and-feel-react/
>     |-- slash-css/
>     |-- slash-react/
> 
> on reste sur une structure à plat des dossiers mais comme le nom du design-system et en préfix cela permet une meilleur visibilité.

Cela faisait sens de grouper les packages css et react au sein d'un répertoire look-and-feel dès lors que ces packages ont un workflow et cycle de vie qu'ils ne partagent pas avec les packages de Slash. Mais on peut faire mettre la structure au même niveau si tu y tiens.

Reply:
C'est juste une question de préférence après on peut remonter d'une arbo package n'as pas vraiment de sens on pour pourrais faire comme ceci

```
/
|-- look-and-feel/
    |-- css/
    |-- react/
|-- slash/
    |-- css/
    |-- react/
```

comme cela on a la séparation franche, sa coller avec le sample

après je suis d'accord avec le reste.

Reply:
```
/
|-- look-and-feel/
    |-- css/
    |-- react/
|-- slash/
    |-- css/
    |-- react/
```
Plus simple et le plus clair effectivement, go pour moi.

Reply:
@PeioDeLaVega Je vois que tu as ramener sur
```
/
|-- packages/
    |-- look-and-feel-css/
    |-- look-and-feel-react/
```
Moi cela me va, comme dit plus haut j'aime pas trop qu'il y en trop d'arborescence pour les différents packages.

Par contre tu n'a fait que de l'ajout sans aucune modification des
```
/
|-- packages/
    |-- css/
    |-- react/
```
est une volonté pour permettre des test les workflow ou ne vaudrait il pas mieux tous séparer d'un coup ?